test/test_jit_fuser.py

```diff
@@ -282,6 +282,22 @@ def funcOptMax(a, b):
             graph = backward_graph(s)
             self.assertAllFused(graph)
 
+    @unittest.skipIf(IS_WINDOWS, "NYI: fuser support for Windows")
+    @unittest.skipIf(not RUN_CUDA, "fuser requires CUDA")
+    @skipIfRocm
+    def test_dropout(self):
+        def func(x):
+            x = torch.nn.functional.dropout(x)
+            return torch.nn.functional.relu(x)
+
+        a = torch.randn(4, 4, dtype=torch.float, device='cuda', requires_grad=True)
+        s = torch.jit.script(func, (a,))
```

Comment:
Do the `torch.jit.script` take an example input as argument? I thought it was only trace that did.

Reply:
It does not have to, but it can.